From: Bastian Germann <bage@debian.org>
Date: Thu, 30 Jul 2020 22:24:10 +0200
Subject: Install versioned shared library

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -131,7 +131,7 @@
 	install -m 644 build/release/libmujs.a $(DESTDIR)$(libdir)
 
 install-shared: install-common build/release/libmujs.$(SO)
-	install -m 755 build/release/libmujs.$(SO) $(DESTDIR)$(libdir)
+	install -m 755 build/release/libmujs.$(SO) $(DESTDIR)$(libdir)/libmujs.so.3
 
 install: install-static
 
