From: Bastian Germann <bage@debian.org>
Date: Wed, 17 Jun 2020 01:17:04 +0200
Subject: Set the right .pc version

Forwarded: not-needed
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -112,7 +112,7 @@
 	@mkdir -p $(@D)
 	echo > $@ Name: mujs
 	echo >> $@ Description: MuJS embeddable Javascript interpreter
-	echo >> $@ Version: $(VERSION)
+	echo >> $@ Version: $(DEB_VERSION_UPSTREAM)
 	echo >> $@ Cflags: -I$(incdir)
 	echo >> $@ Libs: -L$(libdir) -lmujs
 	echo >> $@ Libs.private: -lm
